setup/steps/run_pg_sleep.go

```diff
@@ -0,0 +1,57 @@
+package steps
+
+import (
+	"fmt"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/guregu/null"
+	s "github.com/pganalyze/collector/setup/state"
+)
+
+var RunPgSleep = &s.Step{
+	Description: "Run a pg_sleep command to confirm everything is working",
+	Check: func(state *s.SetupState) (bool, error) {
+		needsSleep := (state.Inputs.SkipLogInsights.Valid && !state.Inputs.SkipLogInsights.Bool) ||
+			(state.Inputs.SkipAutomatedExplain.Valid && !state.Inputs.SkipAutomatedExplain.Bool)
+		return !needsSleep || state.DidPgSleep ||
+			(state.Inputs.SkipPgSleep.Valid && state.Inputs.SkipPgSleep.Bool), nil
+	},
+	Run: func(state *s.SetupState) error {
+		var doPgSleep bool
+		if state.Inputs.Scripted {
+			if state.Inputs.SkipPgSleep.Valid {
```

Comment:
Might be better if we call this `RunPgSleep` (or similar) and flip the logic. That way, we run the pg_sleep test only if that is set (and skip otherwise, i.e. if false or not set, do the skip)